lib/wrapper.js

```diff
@@ -70,7 +70,11 @@ class RequestWrapper {
       logger.info('Canceled request');
     }
     logger.fromError('fatal_error', error, this._getLogParameters());
-    throw new SuiteRequestError(error.message, 500);
+
+    const recoverableErrorCodes = ['ECONNRESET', 'ETIMEDOUT', 'ECONNREFUSED', 'ECONNABORTED'];
+    const code = recoverableErrorCodes.includes(error.code) ? 503 : 500;
+
+    throw new SuiteRequestError(error.message, code, false, error.code);
```

Comment:
Instead of `false`, I'd add an empty object (as the `SuiteRequestError` constructor expects a `request`), or an empty string, if we add `response.body` in `_handleResponse` instead of the whole `request` object.